packages/mui-base/src/SliderUnstyled/useSlider.ts

```diff
@@ -364,11 +364,9 @@ export default function useSlider(parameters: UseSliderParameters) {
   const getFingerNewValue = ({
     finger,
     move = false,
-    values: values2,
   }: {
     finger: { x: number; y: number };
     move?: boolean;
-    values: number[];
```

Comment:
This indirection doesn't seem to make sense, the `values` used when called getFingerNewValue, is the some one available from its upper scope.

Reply:
Nice catch